README.md

```diff
@@ -1 +1,82 @@
-# @moxy/react-modal
\ No newline at end of file
+# react-modal
+
+[![NPM version][npm-image]][npm-url] [![Downloads][downloads-image]][npm-url] [![Build Status][travis-image]][travis-url] [![Coverage Status][codecov-image]][codecov-url] [![Dependency status][david-dm-image]][david-dm-url] [![Dev Dependency status][david-dm-dev-image]][david-dm-dev-url]
```

Comment:
I think you are missing `travis.yml`. However, we could instead start using GitHub actions as it went out of beta in November.